server/serializers.py

```diff
@@ -68,7 +71,29 @@ def _camelize_value(v: Any) -> Any:
         return v
 
 
-JsonizeContext = namedtuple("JsonizeContext", ["user", "session", "locale_id"])
+JsonizeContext = namedtuple(
+    "JsonizeContext", ["user", "session", "locale_id", "localizers"]
```

Comment:
Do we really need to add `localizers` here? Isn't there already a cache on `MessageLocalizer.for_application_messages()` that makes this redundant?